### DISCUSSION TITLE
PatchTool 6.2 8/24

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#214] [Announcements]
相容性更新。

部分「蠻族」用詞因應日文原文更新為「友好部族」，不過尚未全部更新，後續再處理。

檔案：
- FFXIVChnTextPatch-0823.rar：本體。一般而言，下載這個即可。JRE縮小過。

選用：
- rawexd-zhcn-0823.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [第三方4K字體包](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.53/font-4k.rar)：部分解析度設定下會出現字體錯位等情形，請自行照需求使用。解壓後改名為font放到resource資料夾下（取代原資料夾）。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 #90 放到與exe同位置使用。
- backup-0823.rar：乾淨的0a0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/'>PatchTool 6.2 8/24</a>.</em>

**Comment by Manbbb** [comment]
感謝大大的更新~衝啊!!

**Comment by KppKerry** [comment]
謝大大 太神了

**Comment by ooo-Jay** [comment]
不能進入優雷卡, 跟黃金港NPC對話會卡住

**Reply by GpointChen** [reply]
已發現問題，會於下次更新時修正

有需要可先自行替換以下指令碼：
`<hex:02281CFF17>ContentFinderCondition<hex:E8022703>` → `<hex:02281CFF17>ContentFinderCondition<hex:E8022803>`
`<hex:02281CFF17>ContentFinderCondition<hex:E8032703>` → `<hex:02281CFF17>ContentFinderCondition<hex:E8032803>`

**Comment by GpointChen** [comment]
※08/24 09:00 修正優雷卡與副本指令碼錯誤。有需要請重新下載、還原、漢化。

**Comment by r8brnydta0** [comment]
![image](https://user-images.githubusercontent.com/111965020/186368866-2a6eeba3-3070-4a88-b655-33bb07ce42e4.png)

不好意思請問一下，我已經出錯過，我刪掉遊戲重新下載，可是玩一玩又發生這個錯誤，怎麼辦><

**Reply by GpointChen** [reply]
先確認你還原後的檔案是否乾淨，或者可以直接下載release中的backup，解壓後丟到漢化器的backup資料夾下面，然後打開漢化器執行「還原」

然後再次執行漢化

**Comment by opiumei** [comment]
紅蓮節活動會場的NPC對話 UI閃一下後沒反應，重裝了幾次漢化都是一樣 @@

**Reply by GpointChen** [reply]
哪一個NPC？我剛跑了一下紅蓮節的任務沒發現問題

**Reply by opiumei** [reply]
跳跳樂起點那幾位，除了常夏の冒險者，其他人都會閃一下。
我是有先用Back的原始檔再進行csv日文，安裝兩次都這樣，不太確定還會不會是我這邊的問題。

如果大大測試沒事的話，我等等再試看，謝謝<(_ _)>

**Reply by GpointChen** [reply]
我晚點有空再測測看，可能我有NPC沒對話到

**Reply by GpointChen** [reply]
將新任務都跑了一圈確定沒遇到問題
